src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, createContext } from 'react';
+
+import { Stack, StackProps } from '@chakra-ui/react';
+
+export const ProgressContext = createContext<ProgressValues>(null);
+
+export interface ProgressValues {
+  completed: number;
+  total: number;
+  isLoading?: boolean;
+}
```

Comment:
I'd swap the two blocks to declare the ProgressValues before using them.

```suggestion
export interface ProgressValues {
  completed: number;
  total: number;
  isLoading?: boolean;
}

export const ProgressContext = createContext<ProgressValues>(null);
```